### DISCUSSION TITLE
Project name

### DISCUSSION THREAD

**Discussion by escamoteur** [DevsHelpDevs/DevsHelpDevs#11] [Q&A]
Hi, 

if no-one else comes up with a really good other idea I think we go with **devshelpdevs**
and as domain devshelpdevs.org

**Comment by 805702** [comment]
I think the devshelpdevs has already been adopted. It's quite a good and explicit name.

**Comment by 805702** [comment]
is the **.com** tld not available? or why not that? why the choice of the **.org** tld?

**Reply by escamoteur** [reply]
Because .com stands for commercial and we are not a profit oriented group. That's why I think .org is the right one.

**Reply by StephenLadoye** [reply]
.org is perfect for this project
